tests/integration-tests/tests/storage/storage_common.py

```diff
@@ -157,14 +190,14 @@ def _write_user_data(efs_id, random_file_name):
     mount_dir = "/mnt/efs/fs"
     return f"""
         - mkdir -p {mount_dir}
-        - mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport,_netdev "{efs_id}.efs.${{AWS::Region}}.${{AWS::URLSuffix}}:/" {mount_dir}
+        - mount -t efs -o tls,iam {efs_id}:/ {mount_dir}
         - touch {mount_dir}/{random_file_name}
         - umount {mount_dir}
         """  # noqa: E501
 
 
-def test_efs_correctly_mounted(remote_command_executor, mount_dir, tls=False):
-    # TODO: Add iam. The value of the two parameters should be set according to cluster configuration parameters.
+def test_efs_correctly_mounted(remote_command_executor, mount_dir, tls=False, iam=False):
+    # Add iam. The value of the two parameters should be set according to cluster configuration parameters.
```

Comment:
"Add iam" in the comment is no longer needed, right?

Reply:
I'm not sure, but I think it is good to leave as an explanation and artifact of what the purpose of the code is.

Reply:
How are the two words "Add iam" explanatory?

Reply:
It expresses that iam was added to the function signature.

Reply:
That is self-explained by the function signature

Reply:
Sorry, to clarify it indicates that the signature originally only had tls.